docs/source/quick_start.rst

```diff
@@ -0,0 +1,199 @@
+
+  Before installing, you can ``spack spec``  variants to verify the build type.
+
+Serial
+~~~~~~~
+
+::
+
+  spack spec kokkos@4.2 %gcc@10.3.0 +serial cxxstd=20
+
+OpenMP
+~~~~~~
+
+::
+
+  spack spec kokkos@4.2 %gcc@10.3.0 +openmp cxxstd=20
+
+
+CUDA
+~~~~
+
+:: 
+  
+  spack spec / install kokkos@4.2 %gcc@12.2.0 +cuda cuda_arch=70 cxxstd=20 +cuda_relocatable_device_code
+
+
+HIP
+~~~
+
+::
+
+  spack spec / install kokkos@4.2 %gcc@12.2.0 +rocm amdgpu_target=gfx90a cxxstd=20
+
+
+-------------------------------------------
+Building and Linking a Kokkos "Hello World"
+-------------------------------------------
+
+.. note::
+
+  You will need to set ``Kokkos_ROOT``, and also the root directory for you target backend SDK (i.e., ``CUDA_ROOT``, ``ROCM_PATH``).  Please see `Build, Install and Use <https://kokkos.org/kokkos-core-wiki/building.html>`_ for additional details.
+
+|br|
+::
+
+  git clone https://github.com/ajpowelsnl/View
+  cd View
+  mkdir build
+  cd build
+  cmake ../
+
+
+-------------
+Get Involved!
+-------------
+
+Joining the Kokkos Community
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Participate in the lively virtual discussion on the `Kokkos Slack Channel <https://kokkosteam.slack.com>`_.  Here are `sign up details <https://kokkos.org/kokkos-core-wiki/faq.html#faq>`_.
+|br|
+
+Acclerating learning
+~~~~~~~~~~~~~~~~~~~~
+
+Take a deeper dive into Kokkos with over 16 hours of `Tutorials <https://github.com/kokkos/kokkos-tutorials>`_ and `Recorded Lectures <https://github.com/kokkos/kokkos-tutorials/wiki/Kokkos-Lecture-Series>`_.  For in-house workshops and training, please get in touch via Slack (below).
+|br|
+
+Coming to Release Demos
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You have a standing invitation to attend our approximately quarterly release demos to hear what's new, what's gone, and where we're heading in the future.
+|br|
+
+Attending Kokkos Users' Group Meetings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please join us in our annual Kokkos Users' Group Meeting, where we present recent Kokkos work, and you showcase how you use Kokkos.  It's a great opportunity to build community and grow collaboration.
+|br|
+
+
+..
+  *TODO*
+     - Integrate (merged) Quick Start with Cédric's PR:  https://github.com/kokkos/kokkos/pull/6796
+     - Ongoing reconciling with the Julien B. / KUG23- initiated discussion:  https://github.com/kokkos/internal-documents/pull/19
+     - Add `git submodule` "how to" for Kokkos
+     - Add Quick Start to main Kokkos page, such that it is the first thing you encounter on the landing page (kokkos.org)
+     - In V2, put the recipes for the different backends on different pages
+     - Julien B. suggested using github templates for the View "Hello World" example
+     - Nic M.:  CUDA as a CMake language example (using View): cmake -S . -B build -DKokkos_ENABLE_CUDA=ON CMAKE_CUDA_COMPILER=nvcc Kokkos_ENABLE_COMPILE_AS_CMAKE_LANGUAGE=ON [-DCMAKE_BUILD_TYPE=Release]
+
+
+..
+
+.. |br| raw:: html
+
+      <br>
+
```

Comment:
```suggestion
```

Reply:
This html-ism is needed for new lines (`|br|`) in Sphinx.  New lines are needed in certain places because the text is crowded with too many words (towards the bottom).  I would like to leave this html-ism, unless there's a better way to do this in Sphinx.